src/realm/object-store/sync/app.hpp

```diff
@@ -453,17 +493,26 @@ class App : public std::enable_shared_from_this<App>,
 
     /// Request the app metadata information from the server if it has not been processed yet. If
     /// a new hostname is provided, the app metadata will be refreshed using the new hostname.
-    /// @param completion The server response if an error was encountered during the update
-    /// @param new_hostname If provided, the metadata will be requested from this hostname
-    void init_app_metadata(util::UniqueFunction<void(const util::Optional<Response>&)>&& completion,
-                           const util::Optional<std::string>& new_hostname = util::none);
+    /// @param completion The callback that will be provided the pass fail response
```

Comment:
the pass->to pass?
Also, I guess this callback is invoked also when successful right?

Reply:
Yes - I reworded for clarity